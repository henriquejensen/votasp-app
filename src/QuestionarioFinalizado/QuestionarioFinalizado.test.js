import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import QuestionarioFinalizado from './QuestionarioFinalizado';

describe('QuestionarioFinalizado', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <QuestionarioFinalizado /> )))).toMatchSnapshot();
  })
})
