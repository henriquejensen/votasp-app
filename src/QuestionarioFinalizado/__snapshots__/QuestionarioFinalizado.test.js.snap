// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuestionarioFinalizado should match a snapshot 1`] = `
<PageLayout>
  <section>
    <div
      className="container welcome-text"
    >
      <h2>
        <strong>
          Obrigado por responder!
        </strong>
      </h2>
      <p>
        A partir de agora você aparecerá no ranking para todos os eleitores.
      </p>
    </div>
  </section>
</PageLayout>
`;
